mne/preprocessing/xdawn.py

```diff
@@ -62,6 +69,12 @@ def _least_square_evoked(epochs_data, events, tmin, sfreq):
         An concatenated array of toeplitz matrix for each event type.
     """
 
+    if isinstance(solver, string_types):
```

Comment:
Yes, makes sense. 
Waiting with further steps for comments by @alexandrebarachant ...